llama_cpp/llama_cpp.py

```diff
@@ -1185,6 +1185,12 @@ def llama_rope_type(model: llama_model_p, /) -> int:
     ...
 
 
+# LLAMA_API enum llama_pooling_type llama_pooling_type(const struct llama_model * model);
+@ctypes_function("llama_pooling_type", [llama_model_p_ctypes], ctypes.c_int)
+def llama_pooling_type(model: llama_model_p, /) -> int:
+    ...
+
+
```

Comment:
```suggestion
```

I added this function elsewhere in the file to match the llama.h order so we can delete this.